plugins/events-backend/src/service/hub/types.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { EventParams } from '@backstage/plugin-events-node';
+
+export type EventBusStore = {
+  publish(options: {
+    params: EventParams;
```

Comment:
```suggestion
    event: EventParams;
```

Reply:
Feels like this is jumping ahead a bit but alright, updated. Most current usage is `params: EventParams`, although then as a param.